### ISSUE TITLE
Print useful information when buildtest configuration fails

### ISSUE THREAD

**Issue by shahzebsiddiqui** [buildtesters/buildtest#246]
If there is a configuration issue, buildtest just prints the following: 
```
(docs) ssi29@ag-mxg-hulk090> buildtest build -c examples/script/bzip2.yml -d
Buildtest Configuration Check Failed!
Configuration File: /u/users/ssi29/.buildtest/settings.yml failed to validate against schema: /mxg-hpc/users/ssi29/buildtest-framework/buildtest/settings/default.schema.json
```

This is not useful in troubleshooting and turns out this can be fixed when invoking the ``validate`` method. This is addressed in #249

**Comment by vsoch** [comment]
This probably has a dependency on #240 since one or more stages would be run to check it :)

**Comment by vsoch** [comment]
For validation of the file itself, this is already done when it's used and loaded (it will fail on any interaction if invalid).

**Comment by shahzebsiddiqui** [comment]
Issue resolved in #249
